ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java

```diff
@@ -182,10 +184,10 @@ public void getCompletedJob() throws Exception {
 
             final String expectedBeginning = "JobId: " + readJob.getJobId() + " | Status: COMPLETED | Bucket: " + bucketName
                     + " | Type: GET | Priority: HIGH | User Name: spectra | Creation Date: ";
-            final String expectedEnding = " | Total Size: " + objSize + " | Total Transferred: " + objSize;
+            final String expectedEnding = " | Total Size: " + objSize + " | Total Transferred: " ;//TODO add objSize when testing using BP 1.2 is not relevant anymore
 
             assertTrue(getJobResponse.getMessage().startsWith(expectedBeginning));
```

Comment:
Do we still need the startsWith and contains checks if we can check the items directly with the json models?